llama_stack/providers/inline/post_training/torchtune/common/utils.py

```diff
@@ -15,6 +15,8 @@
 
 import torch
 from llama_models.datatypes import Model
+
+from llama_models.llama3.api.datatypes import BaseModel
```

Comment:
Can we merge this? This is breaking all usage besides post training:

```
NameError: name 'BaseModel' is not defined

```

Reply:
Should be `from pydantic import BaseModel`?